files/pt-br/web/api/url/searchparams/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: URL.searchParams
+slug: Web/API/URL/searchParams
+tags:
+  - API
+  - Property
+  - Read-only
+  - Reference
+  - URL
+  - URLSearchParams
+browser-compat: api.URL.searchParams
+---
+
+{{APIRef("URL API")}}
```

Comment:
```suggestion
---
title: URLSearchParams
slug: Web/API/URLSearchParams
page-type: web-api-interface
tags:
  - API
  - Interface
  - Landing
  - Reference
  - URL API
  - URLSearchParams
  - Polyfill
browser-compat: api.URLSearchParams
---
{{ApiRef("URL API")}}
```

Reply:
Oops, looks like these changes got messed up and content builds are failing -- I'll open a follow-up PR to fix this!